sv.c

```diff
@@ -3104,7 +3104,7 @@ Perl_sv_2pv_flags(pTHX_ SV *const sv, STRLEN *const lp, const I32 flags)
 		*--retval = '(';
 
 		retval -= typelen;
-		memcpy(retval, typestr, typelen);
+		memcpy(retval, typestring, typelen);
```

Comment:
not sure the change in this file have any value as part of the `plan9` change?
I see a variable renaming, but I do not think this should belong to these changes?
Any reasons this was changed?

Reply:
@atoomic , look up: https://github.com/Perl/perl5/pull/17891/#pullrequestreview-438441557

Reply:
thanks for the link LGTM